media-video/motion/files/motion.service

```diff
@@ -8,13 +8,13 @@ Type=simple
 ExecStart=/usr/bin/motion -n -c /etc/motion/motion.conf
 StandardError=null
 
-User=motion
+#User=motion
 Group=video
 
 #RuntimeDirectory=motion
 #RuntimeDirectoryMode=0750
 
-WorkingDirectory=/var/lib/motion
+#WorkingDirectory=/var/lib/motion
```

Comment:
Why comment? Just remove if it's not needed.

Reply:
I should actually make a separate r4 version and leave the original as is for motion-4.1.1. That way existing 4.1.1 won't be bothered with any changes and 4.1.1 can be deprecated over time. The new version can set the uid/gid and the working dir properly.